### PR TITLE
Updated README.md to include installation instructions specific to Windows 11.

### PR DESCRIPTION
**Pull request by swstegall**
I encountered difficulties setting this up for Windows 11 with the instructions listed. With a few work-arounds, I was able to get this to work, so I wanted to make a PR to describe the process that made this work. Please let me know if you'd like any revisions.

### CODE REVIEW

**Comment by cathyjf** [comment]
These instructions no longer work because they yield this error:

```
> pnputil /add-driver .\Scream.inf /install         
Microsoft PnP Utility

Adding driver package:  Scream.inf
Failed to add driver package: A required certificate is not within its validity period when verifying against the current system clock or the timestamp in the signed file.

Total driver packages:  1
Added driver packages:  0
```

To work around this, it is necessary to go all the way to clowntown and actually set the clock backward before installing the driver as recommended in https://github.com/duncanthrax/scream/issues/202#issuecomment-1660478233.

However, the `pnputil` installation command now yielded a new and different error:

```
Failed to add driver package: The publisher of an Authenticode(tm) signed catalog has not yet been established as trusted.
```

It turns out this error was caused by the fact that I wasn't using a GUI session of Windows. After running the same command from a GUI session, it finally works.

In conclusion, good guide but the step of changing the clock is sadly necessary now and should be added to the guide.
